variables.tf

```diff
@@ -4,7 +4,12 @@ variable "project_id" {
 }
 
 variable "gke_cluster_name" {
-  type = string
+  type        = string
   description = "GKE cluster name for which to create IAM roles"
 }
 
+
+variable "service_accounts_unique_ids" {
```

Comment:
Shouldn't it have a default value?
Wondering maybe it is possible to make code changes in a way that if ` var service_accounts_unique_ids` is empty, use old permissions approach and if provided use new approach.

My main concern is that this is a "breaking change". If we merge code as is right now and customer haven't pinned this module version, their initial TF/Plan apply fail with error `service_accounts_unique_ids` needs to be provided. And probably it will cause role re-creation which will translate to some time of CAST AI autoscaling downtime.

If it is not possible to make this change, in my opinion, this should be a major version release with updated README.md with "migration" isntructions

Reply:
Yes, it should! 

I have added a default value for `service_accounts_unique_ids` so now if the customers don't have pinned old model versions nothing will break. Additionally, if someone won't specify those ids they will use the default `roles/iam.serviceAccountUser` without any extra conditions - it follows the old approach.

Reply:
Thanks for addressing it. LGTM!